src/Keboola/StorageApi/Client.php

```diff
@@ -1162,7 +1162,7 @@ public function deleteTableAttribute($tableId, $key)
      * @param string $tableId
      * @param string $name
      */
-    public function addTableColumn(string $tableId, string $name, ?array $definition = [], ?string $basetype = null)
+    public function addTableColumn(string $tableId, string $name, ?array $definition = null, ?string $basetype = null)
```

Comment:
Tady to padalo na `Invalid parameters - definition: This field was not expected.`
Protoze kdyz se zavolala `addTableColumn('foo', 'bar')`, tak se `definition` nastavila na `[]`, ale dole se kontroluje `!== null`. Tzn. ze se posilal parametr `definition`.

Z nejakyho duvodu, ve **form-data** se parametr s prazdnym polem vynecha, takze se poslalo: `name=bar`.
Ale pro **JSON** se poslou kompletni data: `{name: bar, definition: []}`. 🤷